conf/gunicorn-dev.conf.py

```diff
@@ -0,0 +1,5 @@
+bind = "0.0.0.0:8001"
+reload = True
+reload_extra_files = "lms/templates"
+timeout = 0
```

Comment:
`timeout = 0` is good for when you're debugging the web process with `pdb`, otherwise Gunicorn restarts the worker after `30 secs` which restarts the process you were in the middle of debugging which is really annoying.